README.md

```diff
@@ -27,255 +20,154 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 | Java / Groovy | Yes                | Yes                |
 | C++           | No                 | Yes                |
 | JavaScript    | No                 | Yes                |
-| gRPC          | -                  | Yes                |
+| Go            | No                 | Yes                |
+| R             | No                 | Yes                |
+
+## Installation
+
+Deephaven Community Core can be used from both server- and client-side APIs. Server-side APIs allow users to connect directly to the Deephaven server and execute commands, write queries, and much more. Client-side APIs allow users to connect to a Deephaven server through a client application.
+
+See the following documentation links for installation instructions and more:
+
+- Python
+  - [Run from Docker](https://deephaven.io/core/docs/tutorials/quickstart/)
+  - [pip-installed](https://deephaven.io/core/docs/tutorials/quickstart/)
+- [Groovy]
+  - [Run from Docker](https://deephaven.io/core/groovy/docs/tutorials/quickstart/)
+- [Python client](https://pypi.org/project/pydeephaven/)
+- [Java client](https://deephaven.io/core/docs/how-to-guides/java-client/)
```

Comment:
I think this whole page should be redone. I don't think it's a good intro to the java client.